rpc/write.go

```diff
@@ -2,51 +2,82 @@ package rpc
 
 import (
 	"context"
-	"errors"
-	"strings"
 )
 
 type BroadcastedInvokeTransaction interface{}
 
 // AddInvokeTransaction estimates the fee for a given Starknet transaction.
-func (provider *Provider) AddInvokeTransaction(ctx context.Context, broadcastedInvoke BroadcastedInvokeTransaction) (*AddInvokeTransactionResponse, error) {
-	// TODO: EntryPointSelector now part of calldata
-	// tx, ok := broadcastedInvoke.(BroadcastedInvokeV0Transaction)
-	// if ok {
-	// 	tx.EntryPointSelector = fmt.Sprintf("0x%x", types.GetSelectorFromName(tx.EntryPointSelector))
-	// 	broadcastedInvoke = tx
-	// }
+func (provider *Provider) AddInvokeTransaction(ctx context.Context, broadcastedInvoke BroadcastedInvokeTransaction) (*AddInvokeTransactionResponse, *RPCError) {
```

Comment:
The removal of Broadcast types is addressed in a separate PR, and I'd rather address it in a merge conflict than make it a dependency

Reply:
I see. There is a [comment](https://github.com/NethermindEth/starknet.go/pull/290#issue-1859560594) in #290 (Remove BroadcastedTxn) to merge it last on account of all the merge conflicts it will generate. Let's do that, we'll merge this PR first and leave #290 for last.